testing/ef-tests/src/models.rs

```diff
@@ -27,8 +28,6 @@ pub struct BlockchainTest {
     pub blocks: Vec<Block>,
     /// The expected post state.
     pub post_state: Option<BTreeMap<Address, Account>>,
-    /// The expected post state merkle root.
-    pub post_state_hash: Option<B256>,
     /// The test pre-state.
```

Comment:
I didn't see any tests with post_state_hash at this level. If we wanted it, we could look in the last block

Reply:
Ah, I was using a newer version of the specs -- Opened up #15809 to update the specs. If that passes, I can make the change here and rebase when merged

Reply:
Closing as it has been merged